HomeTask4/HomeTask4/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace HomeTask4
+{
+    /// <summary>
+    /// We accept from the comand line the path to the file
+    /// </summary>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                ParserForXmlFile str = new ParserForXmlFile(args[0]);
```

Comment:
unclear, OutputtingDataToConsole, and where is parsing? Parse has to be single public method, not private